src/app/data/services/local_storage_service.ts

```diff
@@ -6,17 +6,24 @@ class LocalStorageService {
   private expiryDate:{[key:string]:number} = {};
   private serializer:AjaxContentBlocksHashSerializer = new AjaxContentBlocksHashSerializer();
 
+  localStorage:Storage;
```

Comment:
this should be `private localStorage:Storage;`.

Reply:
are you sure this `Storage`-type exists?

Reply:
Yes, `localStorage` is defined like that in the es6 lib file.